src/pages/patterns/rich-text-editor.mdx

```diff
@@ -0,0 +1,408 @@
+---
+title: Rich-text editor
+description: >
+  A rich-text editor lets a user edit and format text using a "what-you-see-is-what-you-get" (WYSIWYG) editing area.
+---
+
+![Rich-text editor](figma:patterns-rich-text-editor-intro.png)
+
+## Anatomy
+
+A Rich-text editor (RTE) can be used for writing various things such as articles, comments, and messages. The toolbar
+has menus and actions for controlling the appearance of text. It can also be a shortcut for adding content such as
+images, links, and other types of media.
+
+Toolbar actions are designed to look and behave like [Icon buttons](/components/icon-button). They should follow Icon
+buttons as much as possible including focus and hover states.
+
+![Rich-text editor](figma:patterns-rich-text-editor-anatomy.png)
+
+1. [Media button](/components/button#media)
+2. [Media button](/components/button#media) with hidden labels
```

Comment:
Was this intentionally a partial link of the full list item text?
```suggestion
2. [Media button with hidden labels](/components/button#media)
```

Reply:
Yes. Since we don't have an example of a Media button where the labels are hidden, I thought it would be misleading on including the full bullet as the link title.